packages/viz/__tests__/styles/ColorContinuousStyle.spec.ts

```diff
@@ -43,6 +44,45 @@ describe('ColorContinuousStyle', () => {
     });
   });
 
+  describe('Paramters', () => {
+    it('should fails with invalid palette', () => {
+      const style = colorContinuousStyle(FIELD_NAME, {
+        palette: 'unexisting'
+      });
+
+      try {
+        style.getLayerProps(styledLayer);
+      } catch (error) {
+        expect(error).toBeInstanceOf(CartoStylingError);
+      }
+    });
+
+    it('should fails with invalid ranges', () => {
+      const style = colorContinuousStyle(FIELD_NAME, {
+        rangeMin: 1,
+        rangeMax: 1
+      });
+
+      try {
+        style.getLayerProps(styledLayer);
+      } catch (error) {
+        expect(error).toBeInstanceOf(CartoStylingError);
+      }
+    });
+
+    it('should fails with invalid nullColor', () => {
+      const style = colorContinuousStyle(FIELD_NAME, {
+        nullColor: '#'
+      });
+
+      try {
+        style.getLayerProps(styledLayer);
+      } catch (error) {
+        expect(error).toBeInstanceOf(CartoStylingError);
+      }
+    });
+  });
+
   describe('Data validation', () => {
     const opts = {
       palette: ['#f00', '#00f'],
```

Comment:
Not sure about testing so explicitly with chromaScale in this part of the tests, but nevermind. In any case, I'm gonna add a small fix to allow the use of all the colors in the palette, not just first-last, which I think improves quite a bit the result

Reply:
done --> https://toolkit-git-style-helper-validations.carto-frontend.now.sh/examples/style-helpers/color-continuous-style.html